frontend_tests/puppeteer_tests/admin.ts

```diff
@@ -312,25 +312,6 @@ async function test_organization_profile(page: Page): Promise<void> {
     await page.waitForSelector(gravatar_selctor, {visible: true});
 }
 
-async function submit_default_user_settings(page: Page): Promise<void> {
```

Comment:
Since the code is already tested in `change_language` in settings.ts of puppeteer tests, I don't think we need to redo the tests here.